css_styles/styles.css

```diff
@@ -587,3 +618,363 @@ footer {
 .footerbtns {
   display: flex;
 }
+
+/* Desktop version */
+
+@media screen and (min-width: 768px) {
+  .mymenu {
+    visibility: hidden;
+  }
+
+  #myheader {
+    background-color: #3e3c3c;
+  }
+
+  body {
+    overflow-x: hidden;
+  }
+
+  .nav-list {
+    display: flex;
+    margin-right: 206px;
+    margin-top: 26px;
+  }
+
+  .nav-list li {
+    list-style: none;
+    padding-right: 32px;
+    font-size: 15px;
+    font-weight: 600;
+    line-height: 20px;
+    font-family: "Inter", sans-serif;
+  }
+
+  .nav-list li a {
+    color: #fff;
+  }
+
+  .nav-list li a:hover {
+    color: grey;
+  }
+
+  .mylogo {
+    margin-left: 154px;
+  }
+
+  .mybiopage {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: start;
+    height: 870px;
+    width: 100%;
+    left: 0;
+    top: 0;
+    background-image: url(/images/Group-desktop.png);
+    background-repeat: no-repeat;
+    background-position: right;
+    background-color: #1c1a19;
+  }
+
+  .socialbtns-main {
+    height: 100%;
+    justify-items: center;
+    padding-top: 30px;
+  }
+
+  .socialbtns-main ul {
+    display: flex;
+    flex-direction: column;
+    height: 300px;
+  }
+
+  .mybio {
+    padding-left: 75px;
+    width: 1440px;
+    left: 0;
+    top: 0;
+  }
+
+  .mybio h1 {
+    font-family: "Crete Round", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 78px;
+    line-height: 82px;
+    width: 470px;
+    height: 160px;
+    top: 192px;
+  }
+
+  .mybio h2 {
+    width: 566px;
+    height: 24px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+    display: flex;
+    align-items: center;
+  }
+
+  .mybio p {
+    width: 566px;
+    height: 123px;
+    font-family: "Inter", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+    flex: none;
+    order: 2;
+    flex-grow: 0;
+  }
+
+  #myworks {
+    background-color: #fff;
+  }
+
+  .myworks {
+    grid-template-columns: 1fr 1fr;
+    padding: 0;
+    background-color: #fff;
+  }
+
```

Comment:
> - [ ] Great job on making the requested changes as pointed out by the previous reviewers.👍 However, I noticed that some paddings are missing which has led to a misalignment in the design. This causes inconsistencies in the layout and compromises the overall visual appeal and user experience of the website. Therefore, it is important to apply the paddings to ensure that the design is consistent and aligned with the Figma design. Therefore, I kindly request that you add the missing paddings to align the design properly.👍
> 
> Figma design👇:
> 
> ![multi-figma](https://user-images.githubusercontent.com/40009354/236642510-16ae9ab4-0676-46ab-872f-4e522c02ad22.png)

> 
> Your design👇:
> 
> 
> ![multi](https://user-images.githubusercontent.com/40009354/236642523-da443159-1406-452d-b329-b911780bece2.png)

>

Reply:
I have implemented the changes as per your kind suggestions sir.